ui/app/styles/components/exec.scss

```diff
@@ -0,0 +1,146 @@
+.tree-and-terminal {
+  display: flex;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 3.5rem; // nav.navbar.is-popup height
+  bottom: 0;
+
+  .terminal-container {
+    flex-grow: 1;
+    background: black;
+    padding: 16px;
+    height: 100%;
+    position: relative;
+
+    .terminal {
+      height: 100%;
+
+      .xterm .xterm-viewport {
+        overflow-y: auto;
+      }
+    }
+  }
+}
+
+.task-group-tree {
+  background-color: $ui-gray-900;
+  color: white;
+  padding: 16px;
+  width: 200px;
+  flex-shrink: 0;
+  overflow-y: auto;
+
+  .title {
+    text-transform: uppercase;
+    color: $grey-lighter;
+    font-size: 11px;
+  }
+
+  .icon {
+    color: $ui-gray-500;
+  }
+
+  .toggle-button {
+    position: relative;
+    background: transparent;
+    border: 0;
+    color: white;
+    font-size: inherit;
+    line-height: 1.5;
+    width: 100%;
+    text-align: left;
+    overflow-wrap: break-word;
+    padding: 6px 0 5px 17px;
+
+    .icon {
+      position: absolute;
+      left: 0;
+      padding: 3px 3px 0 0;
+      margin-left: -3px;
+    }
+  }
+
+  .task-list {
+    .task-item {
+      padding: 0 8px 0 19px;
+
+      color: white;
+      text-decoration: none;
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+
+      .border-and-label {
+        display: flex;
+        align-items: center;
+        height: 100%;
+        width: 100%;
+        position: relative;
+      }
+
+      .border {
+        position: absolute;
+        border-left: 1px solid $ui-gray-700;
+        height: 100%;
+      }
+
+      .active {
```

Comment:
Also not a big deal, but state classes should be formatted as `is-active`. This is inherited from Bulma despite Ember not being too keen on it.

Reply:
ah this is a class name for the little SVG circle that shows when a task has been selected vs a link style, I could rename to make that clear?

Reply:
I eventually got to that code and saw that 😄 It still seems like a state though in which case the convention still applies. Unless the circle is being named `active`, in which case I rescind my comment.